scripts/docker-integration-tests/prometheus/m3coordinator.yml

```diff
@@ -41,5 +41,15 @@ clusters:
       writeConsistencyLevel: majority
       readConsistencyLevel: unstrict_majority
 
+downsample:
+  rules:
+    mappingRules:
+    - filter: __m3_type__:gauge
+      name: aggregate gauges
+      aggregations: ["Last"]
+      storagePolicies:
+      - resolution: 15s
+        retention: 10h
```

Comment:
Hm, shouldn't we auto apply this? Why is this required in the integration tests?

It will be backwards incompatible if we require to add this rule in place of the automapping rules here before.

Reply:
Yeah, shouldn;t need this.. let me remove.. this change was from a previous version of the diff where I had removed auto mapping rules..